catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -271,6 +270,45 @@ protected void correctJdbcTableFields(
     } catch (SQLException e) {
       throw exceptionMapper.toGravitinoException(e);
     }
+
+    getTableStatus(connection, databaseName, tableName);
+  }
+
+  protected void getTableStatus(Connection connection, String databaseName, String tableName) {
+    // sql is `SHOW ALTER TABLE COLUMN WHERE TableName = 'test_table'`
+    // database name must be specified in connection, so the SQL do not need to specify database
+    // name
+    String sql = String.format("SHOW ALTER TABLE COLUMN WHERE TableName = '%s' limit 1", tableName);
```

Comment:
Can alter table operation be executed in serialization?

Reply:
NO, as metion in https://doris.apache.org/docs/1.2/advanced/alter-table/schema-change/
```
Because the Schema Chanage job is an asynchronous operation, only one Schema chanage job can be performed on the same table at the same time. To check the operation status of the job, you can use the following command
```